config/webpack/webpack-base.js

```diff
@@ -38,15 +38,31 @@ module.exports = {
       options: {
         failOnError: true
       }
+    }, {
+      test: /\.worker\.js$/,
+      use: {
+        loader: 'worker-loader',
+        options: { inline: true, fallback: false }
+      }
     }, {
       test: /\.js$/,
-      exclude: /(node_modules)/,
+      include: /(codecs)/,
```

Comment:
I couldn't find this documented anywhere, and you have the same pattern in `exclude`?

Reply:
I don't think it matters. https://webpack.js.org/configuration/module/#ruleinclude